pdbeccdutils/core/ccd_reader.py

```diff
@@ -395,7 +395,7 @@ def _parse_pdb_properties(cif_block):
     properties = None
     if "_chem_comp." in cif_block.get_mmcif_category_names():
         mod_date = cif_block.find_value("_chem_comp.pdbx_modified_date")
-        if not mod_date:
+        if mod_date == "?":
```

Comment:
I think gemmi has a cif.is_null method (https://gemmi.readthedocs.io/en/latest/cif.html#dom-and-sax) that you can use to test nullity rather than hardcode.

Reply:
done